containers/sfdx-project/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "SFDX in Docker",
```

Comment:
Should this match the template?  **Salesforce Project**

Reply:
Yeah, probably "Salesforce Project" is good. 
Let's also change https://github.com/forcedotcom/vscode-remote-try-sfdx/blob/master/.devcontainer/devcontainer.json